pkgs/development/python-modules/skorch/default.nix

```diff
@@ -72,11 +72,15 @@ buildPythonPackage rec {
       # tries to import `transformers` and download HuggingFace data
       "skorch/tests/test_hf.py"
     ]
-    ++ lib.optionals (stdenv.hostPlatform.system != "x86_64-linux") [
-      # torch.distributed is disabled by default in darwin
-      # aarch64-linux also failed these tests
-      "skorch/tests/test_history.py"
-    ];
+    ++ lib.optionals
+      (stdenv.hostPlatform.system == "x86_64-linux" || stdenv.hostPlatform.system == "aarch64-darwin")
```

Comment:
A bit hacky but I can't think of any improvements right now other than disabling the test completely or making changes upstream.